internals/secrethub/audit.go

```diff
@@ -150,24 +329,180 @@ func (cmd *AuditCommand) iterAndAuditTable() (secrethub.AuditEventIterator, audi
 	return nil, nil, ErrNoValidRepoOrSecretPath
 }
 
+type pager interface {
+	io.WriteCloser
+	IsClosed() bool
+}
+
+// newPaginatedWriter runs the terminal pager configured in the OS environment
+// and returns a writer to its standard input.
```

Comment:
I meant a writer to the standard input of the terminal pager command. Maybe mentioning explicitly that the terminal pager is a command and that the writer is piped to it would make it more intuitive. I didn't mention that since I tried to avoid mentioning how the writer works and only explain what it does.